src/libraries/System.Text.Json/src/System/Text/Json/ThrowHelper.cs

```diff
@@ -4,6 +4,7 @@
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
 using System.Runtime.CompilerServices;
+using static System.Text.Json.Utf8JsonWriter;
```

Comment:
Is this accidental?

Reply:
`EnclosingContainerType` is a nested type in `Utf8JsonWriter` so the import is needed. Outside of exception throwing, it should be functionally private to the writer so I just opted to leave it nested instead of moving it out into its own top level type.